build-setup/macos.md

```diff
@@ -62,6 +63,14 @@ xcode-select --install
 
 at the command prompt.
 
+If (and only if) you are using Mojave, you must install the SDK headers from `/Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg`. You can do this by running:
```

Comment:
This isn't true any more.  I fixed the underlying problem in #669 (and removed something very similar to what you've just added from this file).